Classes/InCallViewControllerNew.m

```diff
@@ -243,6 +243,7 @@ - (void)callUpdate:(LinphoneCall *)call state:(LinphoneCallState)state animated:
             //            NSAssert(0, @"LinphoneCallOutgoingInit: Just need to check this state");
             break;
         }
+        case LinphoneCallOutgoingRinging:
```

Comment:
Yes. There are sanity checks to prevent duplicate timers. This is the only way to get the timer to appear on iPhone 6.